ipc/cli/src/commands/checkpoint/bottomup_bundles.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! List bottom up bundles
+
+use std::fmt::Debug;
+use std::str::FromStr;
+
+use async_trait::async_trait;
+use clap::Args;
+use fvm_shared::clock::ChainEpoch;
+use ipc_sdk::subnet_id::SubnetID;
+
+use crate::commands::get_ipc_provider;
+use crate::{CommandLineHandler, GlobalArguments};
+
+/// The command to get bottom up bundles at height.
+pub(crate) struct GetBottomUpBundles;
+
+#[async_trait]
+impl CommandLineHandler for GetBottomUpBundles {
+    type Arguments = GetBottomUpBundlesArgs;
+
+    async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("get bottom up bundles with args: {:?}", arguments);
+
+        let provider = get_ipc_provider(global)?;
+        let subnet = SubnetID::from_str(&arguments.subnet)?;
+
+        for h in arguments.from_epoch..=arguments.to_epoch {
+            let bundle = provider.get_bottom_up_bundle(&subnet, h).await?;
+            println!("{bundle:?}");
```

Comment:
Would you mind making the print a bit prettier like I did for top-down cross-messages? The debug output is quite hard to read: https://github.com/consensus-shipyard/ipc/blob/65e92a324e4c3818ceeee90f1416d10d8ad9a107/ipc/cli/src/commands/crossmsg/topdown_cross.rs#L44